api_end_end_testing/src/test/java/tds/user/api/tests/CreateUserTest.java

```diff
@@ -0,0 +1,247 @@
+package tds.user.api.tests;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.jayway.restassured.RestAssured.*;
+
+import com.jayway.restassured.RestAssured;
+import com.jayway.restassured.response.Header;
+import com.jayway.restassured.http.ContentType;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.testng.annotations.Test;
+import tds.user.api.model.UserInfo;
+import tds.user.api.model.RoleAssociation;
+
+/*
+    Authenticate user
+    Testing HTTP POST of https://sso-deployment.sbtds.org/auth/oauth2/access_token?realm=/sbac
+    and success status 200
+ */
+public class CreateUserTest extends BaseUri{
+    String accessToken = null;
+    String email = null;
+
+    @Test
+    public void getAuthenticationCode() {
+        RestAssured.baseURI = authenticateURI;
+
+        Header header = new Header("Content-Type", "application/x-www-form-urlencoded");
+
+        accessToken = given()
+            .contentType("application/x-www-form-urlencoded")
+            .header(header)
```

Comment:
looks like header is not needed since it is set via the contentType() method.  at least it seems like that would be the case.